File: cheer-creator/CheerCreator.c
#include <stdio.h>

int main() {
    int yards;
    scanf("%d", &yards);
    
    if (yards > 10) puts("High Five");
    else if (yards < 1) puts("shh");
    else while (yards--) printf("Ra!");
    
    return 0;
}

File: cheer-creator/CheerCreator.java
import java.util.Scanner;

public class Program
{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int yards = input.nextInt();

        if (yards > 10) 
            System.out.print("High Five");
        else if (yards < 1) 
            System.out.print("shh");
        else while (yards-- > 0) 
            System.out.print("Ra!");
    }
}
